services/catalog/src/simcore_service_catalog/core/settings.py

```diff
@@ -24,15 +24,7 @@ def base_url(self) -> str:
         return f"http://{self.DIRECTOR_HOST}:{self.DIRECTOR_PORT}/{self.DIRECTOR_VTAG}"
 
 
-_DEFAULT_SERVICE_RESOURCES = ServiceResources.parse_obj(
-    {
-        "CPU": {"limit": 0.1, "reservation": 0.1},
-        "RAM": {
-            "limit": ByteSize(2 * 1024**3),
-            "reservation": ByteSize(2 * 1024**3),
-        },
-    }
-)
+_DEFAULT_RESOURCES = Resources.parse_obj(DEFAULT_RESOURCES)
```

Comment:
I would have left here defaults ... and eventually use env vars to reconfigure them via configuration ...

Reply:
actually there is an ENV for that: ```CATALOG_SERVICES_DEFAULT_RESOURCE``` I forgot to put it in the ```.env-devel ``` but now it's done